CMakeLists.txt

```diff
@@ -110,6 +110,7 @@ option(DISABLE_GOOGLE_BENCHMARK "Disable Google Benchmarks support." ON)
 option(DISABLE_HTTP "Disable HTTP support.")
 option(DISABLE_NETWORK "Disable multiplayer functionality. Mainly for testing.")
 option(DISABLE_TTF "Disable support for TTF provided by freetype2.")
+option(DISABLE_VERSION_CHECKER "Disable the check for newer versions.")
```

Comment:
The option alone is not enough, this will not set any preprocessor definitions on its own.

Reply:
Does it need to? It seems to me that that is the responsibility of the packagers of the downstream distros.

Reply:
It does if you want to use `!defined(DISABLE_VERSION_CHECKER)` on the C++ side, right.

Reply:
Ah right, I think I understand. It needs a block along these lines, right?

```CMake
if (DISABLE_TTF)
    target_compile_options(libopenrct2 PUBLIC -DNO_TTF)
endif ()
```

Reply:
Thank you! I'm not familiar with CMake so didn't know that, it should be good now.